### ISSUE TITLE
Menu bar disabled in Emacs 25 on OSX

### ISSUE THREAD

**Issue by bixuanzju** [syl20bnr/spacemacs#5508]
#### Description
I just switched to Emacs 25 on OSX, found the menu bar is disabled. This is not the case when I used Emacs 24.5.

#### Reproduction guide
- Start Emacs (version 25)

*Observed behaviour:*
Menu bar is gone

*Expected behaviour:*
Menu bar should be there

#### System Info
- OS: darwin
- Emacs: 25.0.92.1
- Spacemacs: 0.105.14
- Spacemacs branch: develop (rev. 82fdd9a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(spacemacs-helm auto-completion emacs-lisp
                (git :variables git-magit-status-fullscreen t)
                markdown org spell-checking syntax-checking version-control osx colors idris haskell
                (latex :variables latex-build-command "LatexMk")
                (evil-snipe :variables evil-snipe-enable-alternate-f-and-t-behaviors t)
                ocaml proof-general)
```

I did a little research, found the culprit was that the value of `window-system` was changed to `ns` instead of `mac` in Emacs 25.

**Comment by d12frosted** [comment]
Yes it's disabled indeed. But you can enable it by calling `(menu-bar-mode 1)`.

**Comment by d12frosted** [comment]
P. S. I am not sure where it's disabled. By default it's enabled. 

```
(defun spacemacs//removes-gui-elements ()
  "Remove the menu bar, tool bar and scroll bars."
  ;; removes the GUI elements
  (unless (eq window-system 'mac)
    (when (and (fboundp 'menu-bar-mode) (not (eq menu-bar-mode -1)))
      (menu-bar-mode -1)))
  (when (and (fboundp 'scroll-bar-mode) (not (eq scroll-bar-mode -1)))
    (scroll-bar-mode -1))
  (when (and (fboundp 'tool-bar-mode) (not (eq tool-bar-mode -1)))
    (tool-bar-mode -1))
  ;; tooltips in echo-aera
  (when (and (fboundp 'tooltip-mode) (not (eq tooltip-mode -1)))
    (tooltip-mode -1)))
```

And this should be OK. Am I missing something?

**Comment by d12frosted** [comment]
> I did a little research, found the culprit was that the value of window-system was changed to ns instead of mac in Emacs 25.

Ah. Good point.

**Comment by Karunamon** [comment]
This also happens on a fresh install of 24.5.1. A `M-: (menu-bar-mode 1)` restores the options.
![screen shot 2016-07-25 at 09 43 45](https://cloud.githubusercontent.com/assets/562413/17107423/5f2faefa-524c-11e6-996c-08ed761f650d.png)

**Comment by bixuanzju** [comment]
@Karunamon I also use 24.5.1, are you on the develop branch?

**Comment by d12frosted** [comment]
@Karunamon are you sure that you don't have any custom configurations that might disable it? Also, sharing output of `SPC h d s` might be useful.

**Comment by Karunamon** [comment]
@bixuanzju Nope - I never use the develop branch. (I'm new enough to emacs that trying to troubleshoot bugs would making learning much harder). I did the clone using the command on the front webpage.

@d12frosted Also nope - when I said this was on a fresh install, it was done by deleting my `~/.spacemacs` and `~/.emacs.d` - there should be no other configs with an influence on the system.

This is a dance I've become quite familiar with as I play around with things, break it, and then reinstall to get back to a known baseline (usually after installing some package that louses up the key bindings or similar) - with that in mind, *the timing on this is strange*. The menubar only started going away some time last week after one of these reinstalls.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
